### DISCUSSION TITLE
[논문 리뷰] NeRF with Real world

### DISCUSSION THREAD

**Discussion by WongiPark0628** [Pseudo-Lab/9th-builder#7] [plan]
### 스터디 소개
**"리처드 파인만(Feynman)"**
**What I cannot create, I do not understand.**

안녕하세요. 저희가 진행할 스터디는 **Real-World scene을 NeRF에 적용하는 경우, 발생하는 문제들을 다룬 논문**들을 살펴보고자 합니다.
관련 키워드로는 Appearance embedding, transient issue, 그리고 floater등이 있습니다. 그리고,
관련 논문으로는 [NeRF-in-the-wild](https://arxiv.org/abs/2008.02268), [Ha-NeRF](https://rover-xingyu.github.io/Ha-NeRF/), 그리고 [CR-NeRF](https://arxiv.org/abs/2307.08093), 등이 있습니다.

![Making_GIF](https://github.com/user-attachments/assets/5c7d763a-7524-4d07-a1ff-8450ff1a2930)

저희 **스터디는 밀도 있고, 심도 있는 이해를 목표**로 진행할 예정입니다.
2가지 논문에 대해서 매주 한번씩 깊게 토의를 진행하고, 코드에 대해서 면밀히 함께 리뷰하며 진행할 예정입니다. 
대부분의 스터디원 각자가 역할을 맡아, 역할에 대해서 준비하고, 지속적인 의사소통을 하고자 합니다.
자세한 내용은 하단의 학습활동을 참고해주세요..!

${\textsf{\color{red}* NeRF, 3D-GS에 대한 기본적인 이해가 필요하며, 적극적으로 의사소통 하실 분을 선호합니다.}}$
${\textsf{\color{red}*  캠(CAM)을 필수로 켜야 합니다.}}$

### 스터디 목표
1. Real-World에서 NeRF, 3D-GS를 적용하는 Task에 대한 심도 있고, 밀도 높은 이해
2. 의사소통 하며, 쉽게 설명하고, 설득하는 능력 향상

### 스터디 내용
- NeRF에 대한 심도 있는 지식 쌓기 (Appearance embedding, transient issue, 그리고 floater 등)
- 코드 리뷰 및 논문 이해

## 모집 대상
- **적극적은 의사소통 및 도전정신을 가지신 분**
- **NeRF에 대해서 읽어보고, 테스트 해보신 분**
- **건설적인 의사소통을 지향하는 분**
- **코드 읽는데 큰 부담 없으신 분**

${\textsf{\color{red}* NeRF, 3D-GS에 대한 기본적인 이해가 필요하며, 적극적으로 건설적인 의사소통 하실 분을 선호합니다.}}$
${\textsf{\color{red}*  캠(CAM)을 필수로 켜야 합니다.}}$

**모집 인원**
- 3-5명

### 첫 모임 날짜 & 시간 & 장소
- 날짜: 9월 10일 (화요일)
- 시간: 오후 9시
- 장소: [#ROOM-YL](https://discord.gg/nbpAWKUm)

** 매주 화요일 저녁 9시에 정기 모임이 진행됩니다. **

## 학습 활동 및 활동 방법
- 본 스터디는, NeRF를 Real world에 적용하였을 때, 발생하는 문제점들을 다루고, 추후 연구로 확장될 수 있도록 심도 있는 이해를 목표로 하고 있습니다. 스터디에서는, 논문에 대한 리뷰 및 코드 리뷰를 목표로 하고 있습니다. 구체적으로는 팀원을 random으로 2명씩 묶어 논문 리뷰 및 코드 리뷰를 매주 준비합니다. 이후, 발표를 한 후, 청중(스터디 참여자)들은 최소 2개이상 반드시 질문을 해야 합니다.
** (만약 6명이 되는 경우, 총 3팀으로 묶이게 되는데 그러한 경우 로테이션을 하도록 하겠습니다.)** 

### 학습 자료
  - PPT 혹은 블로깅 (청중이 이해할 수 있도록 준비해주셔야 합니다.)

### 스터디 활동 방법
💡 **오픈 아카데미** 형태(=청강 가능)로 진행하며, 발표자료는 가짜연구소를 통해 **공개적으로 배포**.
    
1. **진행 & 발표**
     - ${\textsf{\color{red}NeRF, 3D-GS에 대한 기본적인 이해가 필요하며, 매일 적극적으로 의사소통 하실 분을 선호합니다.}}$
     - ${\textsf{\color{red}캠(CAM)을 필수로 켜야 합니다.}}$
     - ${\textsf{\color{red}약속을 지켜주시기 바랍니다. (2회 이상 어길 시 제명)}}$
    
## 주차별 목표와 학습 내용
💡 주차별 진행계획은 러너(스터디원)분들과 협의에 의해 변동될 수 있습니다.
     (하단의 논문들의 목차는 변경될 수 있습니다.)

| 날짜 | 내용 | 발표자 | 
| ----- | ----- | -------- |
| 2024/09/03 | OT (스터디 소개, 방향 논의, Ice breaking), &  Intro(NeRF, 3D-GS)| [박원기]() |
| 2024/09/10 | [NeRF in the wild](https://nerf-w.github.io/), [Ha-NeRF](https://rover-xingyu.github.io/Ha-NeRF/) | [박원기]() |
| 2024/09/17 |  [SF-NeRF](https://arxiv.org/abs/2303.03966), [CR-NeRF](https://github.com/YifYang993/CR-NeRF-PyTorch) | [박원기]() |
| 2024/09/24 | [NeRF-MS](https://nerf-ms.github.io/), [NeRF-OSR](https://github.com/r00tman/NeRF-OSR)| [박원기]() |
| 2024/10/01 | [Robust-NeRF](https://robustnerf.github.io/)|  [박원기]() |
| 2024/10/08 | [NeRF-On-the-go](https://rwn17.github.io/nerf-on-the-go/)| [박원기]() |
| 2024/10/22 | 휴강 | [박원기]() |
| 2024/10/29 | 추후 업데이트 예정 | [박원기]() |

**Comment by thrcle** [comment]
정리가 깔끔하게 잘 되었네요👍 화이팅입니다!!

**Reply by lynnminn** [reply]
NeRF라니 ..!! 엄청 어려워 보입니다 ㅎㅎ 대단하네요 :) 화이팅하세요 !!!
